flake.nix

```diff
@@ -58,7 +58,7 @@
             env = [
```

Comment:
@chvp I think these changes accidentally slipped into this PR when rebasing?

Reply:
Right, I was going to make this a separate PR and then I forgot.